BOTS-7194: update documentation to include warning of sbfm ruleset deprication (#8521)
cloudflare/cloudflare-docs: content/bots/_partials/_bfm-ruleset-engine.md

---

Super Bot Fight Mode runs during the `http_request_sbfm` phase of the [Ruleset Engine](/ruleset-engine/about/phases/).

> <Aside type='warning' header='Changes to SBFM Rulesets are incoming'>

Reviewer: ```suggestion
<Aside type='warning' header='Change notice for Super Bot Fight Mode rulesets'>
```

> <Aside type='warning' header='⚠️ Changes to SBFM Rulesets are incoming'>

Reviewer 2: ```suggestion
<Aside type='warning' header='Changes to SBFM Rulesets are incoming'>
```

> We are changing the way that Super Bot Fight Mode rules are configured. **No action is need at this time**. However, be aware that we will be deprecating the ability to modify the `http_request_sbfm` phase via the public API.

Reviewer: ```suggestion
Cloudflare is changing the way you configure Super Bot Fight Mode rules. No action is need at this time. However, Cloudflare will deprecate the ability to modify the `http_request_sbfm` phase via the public Cloudflare API.
```

> We are changing the way that super bot fight mode rules are configured. **No action is need at this time**. However, be aware that we will be deprecating the ability to modify the `http_request_sbfm` phase via the public API.

Reviewer 2: ```suggestion
We are changing the way that Super Bot Fight Mode rules are configured. **No action is need at this time**. However, be aware that we will be deprecating the ability to modify the `http_request_sbfm` phase via the public API.
```
